encoding/test/main.go

```diff
@@ -12,44 +12,15 @@ import (
 	"github.com/Layr-Labs/eigenda/encoding/kzg"
 	"github.com/Layr-Labs/eigenda/encoding/kzg/prover"
 	"github.com/Layr-Labs/eigenda/encoding/kzg/verifier"
-	"github.com/Layr-Labs/eigenda/encoding/rs"
 
 	"github.com/consensys/gnark-crypto/ecc/bn254/fr"
 )
 
+// TODO(samlaf): is anyone using this...? Can we delete it?
```

Comment:
I don't think this is being ran on our tests. I would vote to delete it, might have been useful during the development period of the library.